internal/test/strings.go

```diff
@@ -0,0 +1,29 @@
+package test
+
+import (
+	"strings"
+	"testing"
+
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+// CompareMultipleValues compares comma-separated values, whatever the order is
+func CompareMultipleValues(t *testing.T, value, expected string) {
```

Comment:
Not sure about this function name, or quite what this is for.

Reply:
Yeah… I moved code around, kept the name